src/utils/validationSchema.js

```diff
@@ -0,0 +1,32 @@
+import Joi from 'joi';
+/**
+ * This is the schema definition
+ * for a new userAccount
+ */
+const newUserSchema = Joi.object().keys({
+  fullName: Joi
+    .string()
+    .trim(),
+  username: Joi
+    .string()
+    .alphanum()
+    .lowercase()
+    .trim()
+    .min(8)
+    .required(),
+  email: Joi
+    .string()
+    .lowercase()
+    .trim()
+    .email({ minDomainAtoms: 2 })
+    .required(),
+  password: Joi
+    .string()
+    .alphanum()
+    .min(8)
+    .required(),
+}).options({ stripUnknown: true });
+
+export default {
```

Comment:
lets use `export const newUserSchema`

Reply:
We agreed to use this export default until we have another constant we want to export from inside the module